scripts/install_compiler_linux.sh

```
@@ -40,13 +40,21 @@ touch compiler/version.txt
 
 CURRENT_VERSION=$(< compiler/version.txt)
 
+download () {
+	if [ "$OS" = macos ]; then
+		curl -o $1 -f -L $2
```

Linux also has curl, so why not use curl there too?

Also `-f` "Fail silently (no output at all) on HTTP errors" seems like it would lead to more bug reports.

---

I thought you maybe liked wget better or something, but yeah I can change that

---

So the problem with curl is somehow you can't do the `if [ $? != 0 ]` thing without `-f`, because of what is considered an error. If the document can't be found the server still responds with a 404 for example.
But having `-f -S` has the return code logic while still outputting an error so I will use that.